src/main/java/com/jalasoft/jfc/model/strategy/CommandVideoThumbNail.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.model.strategy;
+
+import com.jalasoft.jfc.model.video.VideoCommand;
+
+/**
+ * This Class contains Command Video ThumbNail.
+ *
+ * @version 0.1 23 Dic 2019.
+ *
+ * @author Oscar Lopez.
+ */
+public class CommandVideoThumbNail implements ICommandStrategy{
+
+    // Content value to rotate.
+    private int commandValue;
+
+    // Content value one second to extract a thumbnail.
```

Comment:
improve this code line to Content value **of** one second to extract a thumbnail